src/resources/js/v2/tickets-commerce.js

```diff
@@ -136,9 +148,42 @@ tribe.tickets.commerce = {};
 		// Bind container based events.
 		obj.bindCheckoutItemDescriptionToggle( $container );
 
+		// Bind loader visibility.
+		obj.bindLoaderEvents();
+
 		$document.trigger( 'afterSetup.tribeTicketsCommerceCheckout', [ $container ] );
 	};
 
+	/**
+	 * Bind loader events.
+	 *
+	 * @since TBD
+	 */
+	obj.bindLoaderEvents = function () {
+		$document.on( obj.customEvents.showLoader, obj.loaderShow );
+		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
+	}
+
+	/**
+	 * Show the loader/spinner.
+	 *
+	 * @since TBD
+	 */
+	obj.loaderShow = function() {
+		const loadClass = obj.selectors.loader;
+		$( loadClass ).removeClass( obj.selectors.hidden );
+	};
+
+	/**
+	 * Hide the loader/spinner.
+	 *
+	 * @since TBD
+	 */
+	obj.loaderHide = function() {
+		const loadClass = obj.selectors.loader;
+		$( loadClass ).addClass( obj.selectors.hidden );
+	};
+
```

Comment:
We already have the implementation of `tickets-loader.js` that we can make use of. Here's the library: https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/js/v2/tickets-loader.js

And here's an example of how this can be used from other places: https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/js/v2/rsvp-manager.js#L178

We need to be sure that the tickets commerce script uses tickets loader as a dependency and then we can make use of the library instead of re-implementing the loader.